CHANGES.md

```diff
@@ -44,6 +44,7 @@ To be released.
  -  Added .NET Core 2.2 as a targeted framework.  [[#209], [#561]]
  -  `TurnClient.AcceptRelayedStreamAsync()` became to ignore disconnected
     connections.  [[#469]]
+ -  `LiteDBStore` became to guarantee atomicity of storing blocks. [[#584]]
```

Comment:
Isn't this a bugfix?

Reply:
We wrote about guaranteeing transaction atomicity as behavioral changes on 0.5.0. but I also think that it's closer to bugfix as you say. so I'll move it to bug fixes section.